islandora_ocr.module

```diff
@@ -7,6 +7,10 @@
 
 // Constant (deprecated).
 define('ISLANDORA_OCR_TESSERACT_REQUIRED_VERSION', '3.02.02');
+// Constants (not deprecated).
+const ISLANDORA_OCR_GENERATE_OCR_RELS_STATEMENT = 'generate_ocr';
+const ISLANDORA_OCR_GENERATE_HOCR_RELS_STATEMENT = 'generate_hocr';
+const ISLANDORA_OCR_UPDATE_7001_TIMESTAMP = 'islandora_ocr_update_7001_generate_ocr_timestamp';
```

Comment:
Where is this used? ISLANDORA_OCR_UPDATE_7001_TIMESTAMP

Reply:
This one yeah... not sure... It [used to be used](https://github.com/Islandora/islandora_ocr/pull/81/commits/d4c4adf45ad4aa101d29dcd8ab9001a078ffeb12), but things were squashed... May be an oversight? @qadan / @willtp87 ?